pyaedt/modules/solutions.py

```diff
@@ -3911,13 +3911,13 @@ def __init__(
         max_frequency="1GHz",
         ray_density=2,
         bounces=5,
-        intrinsics={},
+        intrinsics=None,
     ):
         self.is_creeping_wave = is_creeping_wave
         self._postprocessor = postprocessor
         self._ofield = postprocessor.ofieldsreporter
         self.quantity = quantity
-        self.intrinsics = intrinsics
+        self.intrinsics = [] if intrinsics is None else intrinsics
```

Comment:
```suggestion
        self.intrinsics = {} if intrinsics is None else intrinsics
```

Reply:
@SMoraisAnsys I do agree, based on that if check the only possible option is for self.intrinsics to be a dictionary.
Can we remove the check:
`if isinstance(self.intrinsics, list):` ?
also we have to fix the return type of intrinsicsVar that always returns a string and not a list or dict as stated.
Any comments? @maxcapodi78 @Samuelopez-ansys

Reply:
Yeah, I assume that this should be how to handle it.

Reply:
I agree with all, that intrinsics is difficult because the AEDT API returns different types depending on the case, but I agree, we should standardize.